### PR TITLE
1st branch commit

### PR DESCRIPTION
**Pull request by Tigransahakyan993**

### CODE REVIEW

File: api/router/restaurants.js [hunk 1]
```diff
@@ -2,30 +2,42 @@ const express = require('express');
...
+  requireAuth,
+  initFields,
+  authGuard.roles('owner', 'buyer', 'admin'),
   restaurantsController.getRestaurant
 )
 
 restaurants.post('/',
+  requireAuth,
+  initFields,
+  authGuard.roles('owner', 'buyer', 'admin'),
```
Comment (vladimir2492) [inline api/router/restaurants.js]: Fix

File: api/controller/userController.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    const data = req.body;
+    const user = await serviceDecorator.create(userService, converter, data);
+    writeStatus(res, 200, user);
+  } catch (e) {
+    writeStatus(res, 401, {message: e.message});
+  }
+}
+
+exports.getCurrentUser = async (req, res) => {
+  try {
```
Comment (vladimir2492) [inline api/controller/userController.js:40]: why do we need use try catch??

**Review by vladimir2492 — APPROVED** [review]

File: api/controller/userController.js (hunk 2 above)
Comment (Tigransahakyan993) [reply api/controller/userController.js:40]: fixed

File: api/router/restaurants.js (hunk 1 above)
Comment (Tigransahakyan993) [reply api/router/restaurants.js]: fixed

**Pull request merged**